code/game/turfs/simulated/wall/mineral_walls.dm

```diff
@@ -192,6 +192,11 @@
 	icon_state = "map-overspace"
 	fixed_underlay = list("space"=1)
 
+/turf/closed/wall/mineral/titanium/ship
+	icon = 'icons/turf/shuttle.dmi'
+	icon_state = "wall3"
```

Comment:
In game the standard titanium walls are the white shuttle one's. whereas I wanted to use the black syndicate shuttle walls, but I also did not want to give the crew/marine access to the syndi shuttle mineral/metal. As such I simply created a titanium wall subtype that uses the syndi wall sprites.